internal/controller/instaslice_controller.go

```diff
@@ -84,6 +84,12 @@ func (r *InstasliceReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, nil
 	}
 
+	// Pods with scheduling gates other than the InstaSlice gate are not ready to be scheduled and should be ignored
+	if IsPodGatedByOthers(pod) {
```

Comment:
```suggestion
	if isPodGatedByOthers(pod) {
```

If it is only used in this controller I don't think we should export it.

Reply:
done!